content/team/ravin-kumar/contents.lr

```diff
@@ -5,3 +5,14 @@ picture: https://avatars0.githubusercontent.com/u/7213793?s=400&u=a3ce97892d6f56
 github_url: https://github.com/canyon289
 ---
 location: Los Angeles, USA
+---
+blurb: Trying to develop and apply new methodology for a robust and reproducible statistical analysis workflow.
```

Comment:
Isn't this the same blurb that you changed for Oriol? :D

Reply:
This looks like too aggressive copy pasting, one of the blurbs should be deleted for the website to render correctly